### PR TITLE
Added ability to use 'scope' in a form relation field

### PR DESCRIPTION
**Pull request by blackxel**

### CODE REVIEW

File: lang/en/lang.php [hunk 1]
```diff
@@ -250,6 +250,8 @@
         'property_name_from_description' => 'Relation column name to use for displaying a name.',
         'property_relation_select' => 'Select',
         'property_relation_select_description' => 'CONCAT multiple columns together for displaying a name',
+        'property_relation_scope' => 'Scope',
+        'property_relation_scope_description' => 'Specifies a query scope method defined in the related form model to apply to the list query always.',
```
Comment (bennothommo) [inline lang/en/lang.php]: @blackxel I know this line is a direct copy from the docs, but would you mind changing it to this as it reads better:
```suggestion
        'property_relation_scope_description' => 'Specifies a query scope method that's defined in the related form model to always apply to the list query.',
```

Comment (LukeTowers) [reply lang/en/lang.php]: @bennothommo double quotes wrapping or backslash please, this breaks it with the extra apostrophe

Comment (bennothommo) [reply lang/en/lang.php]: Sorry @LukeTowers - early morning brain.

Comment (LukeTowers) [reply lang/en/lang.php]: No worries @bennothommo, that's what the team's for

**Pull request merged**